lib/const.rb

```diff
@@ -120,21 +120,21 @@
 
 CREW_DEST_HOME          = CREW_DEST_DIR + HOME
 CREW_CACHE_DIR          = ENV.fetch('CREW_CACHE_DIR', "#{HOME}/.cache/crewcache")
-CREW_CACHE_BUILD        = ENV['CREW_CACHE_BUILD']
-CREW_CACHE_FAILED_BUILD = ENV['CREW_CACHE_FAILED_BUILD']
+CREW_CACHE_BUILD        = ENV.fetch('CREW_CACHE_BUILD', nil)
+CREW_CACHE_FAILED_BUILD = ENV.fetch('CREW_CACHE_FAILED_BUILD', nil)
 
 # Set CREW_NPROC from environment variable or `nproc`
 CREW_NPROC = ENV.fetch('CREW_NPROC', `nproc`.chomp)
 
 # Set following as boolean if environment variables exist.
-CREW_CACHE_ENABLED                   = !!ENV['CREW_CACHE_ENABLED']&.eql?('1')
-CREW_CONFLICTS_ONLY_ADVISORY         = !!ENV['CREW_CONFLICTS_ONLY_ADVISORY']&.eql?('1')         # or use conflicts_ok
-CREW_DISABLE_ENV_OPTIONS             = !!ENV['CREW_DISABLE_ENV_OPTIONS']&.eql('1')              # or use no_env_options
-CREW_FHS_NONCOMPLIANCE_ONLY_ADVISORY = !!ENV['CREW_FHS_NONCOMPLIANCE_ONLY_ADVISORY']&.eql?('1') # or use no_fhs
-CREW_NOT_COMPRESS                    = !!ENV['CREW_NOT_COMPRESS']&.eql?('1')                    # or use no_compress
-CREW_NOT_LINKS                       = !!ENV['CREW_NOT_LINKS']&.eql?('1')                       # or use no_links
-CREW_NOT_STRIP                       = !!ENV['CREW_NOT_STRIP']&.eql?('1')                       # or use no_strip
-CREW_NOT_SHRINK_ARCHIVE              = !!ENV['CREW_NOT_SHRINK_ARCHIVE']&.eql?('1')              # or use no_shrink
+CREW_CACHE_ENABLED                   = !ENV['CREW_CACHE_ENABLED']&.eql?('1').nil?
```

Comment:
Note that `.eql?('1')` will return `false` when it is not empty and not equal to `1`, adding `!` and `.nil?` will return `true` anyways

Reply:
Can you adjust the logic such that it doesn't trip up rubocop? Or alternately add an exception to the rubocop config so rubocop doesn't trip on the original code?

Reply:
Are we not able to simply fetch these from the environment like so?
```ruby
CREW_CACHE_ENABLED = ENV.fetch('CREW_CACHE_ENABLED', false)
```
As I understand it, this results in `CREW_CACHE_ENABLED` being false by default, and true if it has an environment variable set (given that Ruby only views `false` and `nil` as false, and everything else as true)

Reply:
@Zopolis4 See my comment in #8889:
```
What is wrong with `ENV.fetch` or simply `CREW_BUILD_FROM_SOURCE = ENV['CREW_BUILD_FROM_SOURCE']`?

Some people might be uncomfortable with actually unsetting a variable with unset
(like me :) ), they would probably run something like `VAR=''` to `"unset"` them
(the env variable still exists but being empty).

Also, this prevents unexpected results and misunderstanding,
like `export CREW_BUILD_FROM_SOURCE=false` being evaluated as true
```

@satmandu How about this:
```ruby
CREW_CACHE_ENABLED = ENV.fetch('CREW_CACHE_ENABLED', false).eql?('1')
```

Reply:
Perhaps we could document somewhere that we require variables to be properly unset? It just feels like a lot of extra code for what feels like a boilerplate task.

Reply:
I think we should check and prevent those "edge cases" (not every user will read the docs, they will open an issue first)

However, we can still make a suggestion on it

Reply:
@supechicken Made as much as possible use that format with 
```ruby
CREW_VARIABLE = ENV.fetch('CREW_VARIABLE', '0').eql?('1')
```